src/components/Stayindays.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const  Stayindays = (props) => {
```

Comment:
This isn't a component, but a function.
1. No need to import React, as you're not using JSX here
2. Should probably be in a different folder than `components`